core/templates/dev/head/pages/exploration_player/HintAndSolutionModalService.js

```diff
@@ -65,9 +68,13 @@ oppia.factory('HintAndSolutionModalService', [
             function($scope, $rootScope, $uibModalInstance) {
               $scope.isHint = false;
               var solution = HintsAndSolutionManagerService.displaySolution();
+              var solutionContentId = solution.explanation.getContentId();
+              var contenIdsToAudioTranslations =
+                ExplorationPlayerService.getContenIdsToAudioTranslations(
+                  PlayerPositionService.getCurrentStateName());
               AudioTranslationManagerService.setSecondaryAudioTranslations(
-                solution.explanation.getBindableAudioTranslations(),
-                solution.explanation.getHtml(),
+                contenIdsToAudioTranslations.getBindableAudioTranslations(
```

Comment:
contentIdsToAudioTranslations (or rename to audioAssets)

Reply:
Ah, I have already done this locally. Done!